docs/http-api/swagger.yaml

```diff
@@ -0,0 +1,2496 @@
+        - application/vnd.eventstore.competingatom+xml
+      parameters:
+        - name: stream
+          in: path
+          description: The stream the persistent subscription is on
+          required: true
+          type: string
+        - name: subscription
+          in: path
+          description: The name of the subscription group
+          required: true
+          type: string
+        - name: ids
+          in: query
+          description: The ids of the messages that need to be nacked separated by commas
+          required: false
+          type: string
+        - name: action
+          in: query
+          description: >-
+            <ul><li>**Park** - Don't retry the message, park it until a request
+            is sent to reply the parked messages<li>**Retry** - Retry the
+            message<li>**Skip** - Discard the message<li>**Stop** - Stop the
+            subscription</ul>
+          required: false
+          type: string
+          enum:
+            - Park
+            - Retry
+            - Skip
+            - Stop
+      responses:
+        "200":
+          description: New persistant subscription
+        "400":
+          description: bad input parameter
+  "/subscriptions/{stream}/{subscription}/replayParked":
+    post:
+      tags:
+        - Subscriptions
+      summary: Replay any previously parked messages in a stream
+      description: >-
+        Replay any previously parked messages in a stream that were parked by a
+        negative acknowledgement action.
+      operationId: Replay previously parked messages
+      produces:
+        - application/vnd.eventstore.competingatom+json
+        - application/vnd.eventstore.competingatom+xml
+      parameters:
+        - name: stream
+          in: path
+          description: The stream the persistent subscription is on
+          required: true
+          type: string
+        - name: subscription
+          in: path
+          description: The name of the subscription group
+          required: true
+          type: string
+        - name: stopAt
+          in: query
+          description: The stream event number to stop at.
+          required: false
+          type: long
+      responses:
+        "200":
+          description: OK
+  /subscriptions/restart:
+    post:
+      description: Restart the subscription subsystem on a node.
+      summary: Restart subscriptions
+      tags:
+        - Subscriptions
+      operationId: Restart subscriptions
+      parameters: []
+      responses:
+        "200":
+          description: OK
+        "401":
+          description: Unauthorized
+      security:
+        - basicAuth: []
+  /users:
+    get:
+      tags:
+        - Users
+      summary: Get all users
+      description: Returns all users defined in EventStoreDB.
+      operationId: Get all users
+      produces:
+        - application/json
+        - application/xml
+      responses:
+        "200":
+          description: OK
+        "404":
```

Comment:
```suggestion

```
This operation can't return not found

All operations in the UsersController can return 401 Unauthorized